stelaval/assignment1/Anagram.java

```diff
@@ -0,0 +1,95 @@
+package assignment1;
+
+import java.util.Arrays;
+
+public class Anagram {
+
+	/**
+	 * Determines if one string is an anagram of another string
+	 * 
+	 * @param s1 first string
+	 * 
+	 * @param s2 second string
+	 * 
+	 * @return true if s1 and s2 are anagrams of each other
+	 */
+	static boolean isAnagram(String s1, String s2) {

```

Comment:
API methods should be public. Otherwise they are available only from within your package, so nobody else can use it.